src/routes/kickPlayer.ts

```diff
@@ -0,0 +1,92 @@
+import { getClient } from '../utils/databaseConnection'
+import { rateLimiter } from '../utils/rateLimiter'
+import { celebrate, Joi, Segments } from 'celebrate'
+import { verifyFCMToken } from '../utils/firebase'
+
+import express, { type Router } from 'express'
+const router: Router = express.Router()
+
+router.get(
+  '/kickPlayer',
+  rateLimiter,
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      creatorToken: Joi.string()
+        .required()
+        .min(1)
+        .max(250)
+        .label('creatorToken'),
+      playerToken: Joi.string().required().min(1).max(250).label('playerToken'),
+    }),
+  }),
+  async (req, res) => {
+    if (
+      !(
+        (await verifyFCMToken(req.query.creatorToken)) &&
+        (await verifyFCMToken(req.query.playerToken))
+      ) ||
+      req.query.creatorToken === req.query.playerToken
```

Comment:
I don't think the creator can `kick` himself. He can leave using the `/leaveGame` endpoint